src/www/messages/iw.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
`iw` is deprecated and shouldn't be used:
https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes#he

Don't we already have `hw` for Hebrew?

Reply:
Right, `he` and `iw` are identical.